RxCocoa/iOS/DataSources/RxPickerViewAdapter.swift

```diff
@@ -31,12 +31,12 @@ private class RxPickerViewArrayDataSource<T>: NSObject, UIPickerViewDataSource,
     }
 }
 
-private class RxPickerViewSequenceDataSource<S: Sequence>
+public class RxPickerViewSequenceDataSource<S: Sequence>
```

Comment:
Hi @anonymcek ,

I don't think any of these elements should be `public`. 

Can you please try just removing `private` identifier from `RxPickerViewSequenceDataSource` and from `RxPickerViewArrayDataSource`?